packages/types/templates.ts

```diff
@@ -1,16 +1,31 @@
 import { z } from "zod";
-
 import { ZLegacySurveyQuestions, ZLegacySurveyThankYouCard, ZLegacySurveyWelcomeCard } from "./LegacySurvey";
 import { ZSurveyHiddenFields, ZSurveyQuestions, ZSurveyThankYouCard, ZSurveyWelcomeCard } from "./surveys";
 import { ZUserObjective } from "./user";
 
+export const ZTemplateChannel = z.enum(["Website Survey", "App Survey", "Email Survey", "Link Survey"]);
```

Comment:
Channel and industry will be a central part of the product config so we need a universal values for them.
I think the easiest would be to choose the existing survey types: `link`, `website`, and `app`. So we could use `ZSurveyType` for this.

Reply:
this also makes URL prefilling more consistent